pkg/quota/api/register.go

```diff
@@ -30,6 +30,9 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&kapi.ListOptions{},
 		&ClusterResourceQuota{},
 		&ClusterResourceQuotaList{},
+
+		&kapi.DeleteOptions{},
+		&kapi.ListOptions{},
```

Comment:
> Add versioned watch if you haven't.

This pull puts clusterresourcequota in the legacy API group, which should already have that registered via kube.